app/services/league/controllers/tooltipController.js

```diff
@@ -0,0 +1,42 @@
+const cheerio = require('cheerio');
+const fetch = require('node-fetch');
+const Tooltip = require('../models/tooltipModel');
+
+exports.scrapeWebsite = async (req, res) => {
+    const terms = req.body.terms;
+    
+    for (const term of terms) {
+        const url = `https://www.investopedia.com/terms/${term[0]}/${term}.asp`;
+        const htmlBody = await fetch(url)
+                        .then((res) => res.text());
+        const $ = cheerio.load(htmlBody);
+        $.html();
+        
+        const definition = $('#mntl-sc-block_1-0-1').text();
+
+        const tooltip = new Tooltip({
+            term: term,
+            definition: definition.trim(),
+            url: url,
+        });
+
+        await tooltip.save();
+    }
+
+    res.send('Successfully saved tooltips.');
+}
```

Comment:
Is this function supposed to be if we want to add a list of terms to the database later, after using scrape.js initially? I'm not sure if this is necessary, especially since we need to manually place them anyways. We would probably just edit the scrape.js terms list, so we probably won't need an endpoint for something that has to be manually maintained.

Reply:
Yeah, I had originally created `scrape.js` to test the script but I'm gonna change it so that we don't add any extra API endpoints.